additional_materials.md

```diff
@@ -0,0 +1,7 @@
+# Additional materials
+
+A collection of complementary helpful content
+
+## Talks
+
+- ["Design Patterns in Rust" by Nick Cameron at the PDRust (2016)](https://www.youtube.com/watch?v=Pm_oO0N5B9k)
```

Comment:
```suggestion
- [Design Patterns in Rust](https://www.youtube.com/watch?v=Pm_oO0N5B9k) by Nick Cameron at the PDRust (2016)
```

Reply:
Do you think it's better like this? Anyway, feel free to keep this or your version and then you can merge it!

Reply:
I think I like that your version better. But you need to approve again after a change ;-)